examples/java/BUILD

```diff
@@ -63,3 +81,19 @@ cc_binary(
         "@bazel_tools//tools/jdk:jni",
     ],
 )
+
+cc_binary(
+    name = "native_runfile",
+    srcs = [
+        "com/example/JavaNativeRunfileFuzzTest.cpp",
+        "com/example/JavaNativeRunfileFuzzTest.h",
+    ],
+    data = [
+        "corpus_1.txt",
+    ],
+    linkshared = True,
```

Comment:
Since this attribute appears unusual, perhaps add a small comment explaining why it is needed?

Reply:
I added a comment.

Reply:
Thanks!